programs/gateway/tests/configure_registrar.rs

```diff
@@ -197,7 +196,7 @@ async fn test_configure_registrar_with_invalid_spl_gov_program_id_error(
         .unwrap();
 
     // Assert
-    assert_anchor_err(err, anchor_lang::error::ErrorCode::ConstraintOwner);
+    assert_gateway_err(err, GatewayError::InvalidRealmForRegistrar);
```

Comment:
the reason this has changed because anchor refuses to build if we use the previous assertion/constraint ![attached image](https://github.com/solana-labs/governance-program-library/assets/77497858/ddad4431-60e3-4b81-baf5-9737f311cc03)

and I believe this is due to the registrar having the same name as the registrar account in [here](https://github.com/crypt0miester/governance-program-library/blob/645d81930afbeb056bd3c8545ebbc68b2b32c3b2/programs/gateway/src/instructions/configure_registrar.rs#L23) 

we could either change the name of the [registrar to registrar_account](https://github.com/crypt0miester/governance-program-library/blob/645d81930afbeb056bd3c8545ebbc68b2b32c3b2/programs/gateway/src/instructions/configure_registrar.rs#L13) to be able to use the previous method and the anchor compiler does not complain.